client/apps/edit/components/admin/components/autocomplete_list_metaphysics.jsx

```diff
@@ -18,7 +19,12 @@ export class AutocompleteListMetaphysics extends Component {
     model: PropTypes.string,
     onChangeArticleAction: PropTypes.func,
     placeholder: PropTypes.string,
-    user: PropTypes.object
+    user: PropTypes.object,
+    type: PropTypes.string
+  }
```

Comment:
I know this comes from the legacy code but I find 'select' versus list to be pretty vague, and wonder if it might make sense to use a boolean of `limitOne` to invoke the select component, rather than a type.

Reply:
Yeah, that's a good point. This is a good opportunity to be more explicit! In that case I would like to rename the sub-component too -- what are your thoughts on `AutocompleteSingle` or `AutocompleteSingleList` and then still use `limitOne` for interfacing with the general `AutocompleteMetaphysics` component?

Reply:
`type` might still be a good option here -- if we want to include inline-list, for example.